pkg/kudoctl/cmd/plan/plan_status.go

```diff
@@ -48,6 +48,17 @@ func status(kc *kudo.Client, options *Options, ns string) error {
 		return nil
 	}
 
+	getPhaseStrategy := func(s string) string {
+		for _, plan := range ov.Spec.Plans {
+			for _, phase := range plan.Phases {
+				if string(phase.Name) == s {
+					return string(phase.Strategy)
```

Comment:
here as well

Reply:
If think that if I wanted to return phase.Strategy I'd have to include the v1beta1 package given its type (v1beta1.Ordering).  Seemed simpler just to return it as a string here.

Reply:
I would prefer having the import over a cast

Reply:
@ANeumann82 Fair enough, I guess that's the more 'Go-like' approach 👍